Corona-Warn-App/src/main/res/layout/person_details_qr_card_item.xml

```diff
@@ -52,4 +52,18 @@
         app:layout_constraintTop_toBottomOf="@id/qr_title"
         tools:text="Perobenahme am 31.05.21, 18:01 Uhr" />
 
+
+    <com.google.android.material.button.MaterialButton
+        android:id="@+id/start_validation_check_button"
+        style="@style/Widget.MaterialComponents.Button.OutlinedButton"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="4dp"
+        android:text="@string/start_validation_rule_check_button"
+        android:textColor="@color/colorSecondary"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintStart_toStartOf="@id/qr_subtitle"
+        app:layout_constraintTop_toBottomOf="@+id/qr_subtitle"
+        app:strokeColor="@color/colorSecondary" />
+
```

Comment:
Why is this not a `ProgressLoadingButton`?

Reply:
this button has a different outlined style than the loading button, so far I have not found a way to pass the button `style` from a custom attribute

app:buttonStyle = "@style/buttonPrimary"
app:buttonStyle = "@style/OutlinedStyle"

any ideas are very welcome ? 🙏

Reply:
I will look into that 👍

Reply:
Addressed https://github.com/corona-warn-app/cwa-app-android/pull/3668